apps/package.json

```diff
@@ -184,8 +184,8 @@
     "scratch-vm": "v0.1.0-prerelease.1501105826-prerelease.1501105849",
     "script-loader": "^0.7.0",
     "seedrandom": "2.4.2",
-    "sinon": "^2.1.0",
-    "sinon-chai": "^2.9.0",
+    "sinon": "3",
+    "sinon-chai": "2.13",
```

Comment:
Since yarn locks our dependencies, I think we can start putting looser test dependencies in our package.json.  In general we want to keep up with newer versions if we can!